src/base/bittorrent/session.h

```diff
@@ -626,7 +626,7 @@ namespace BitTorrent
         bool addTorrent_impl(const std::variant<MagnetUri, TorrentInfo> &source, const AddTorrentParams &addTorrentParams);
 
         void updateSeedingLimitTimer();
-        void exportTorrentFile(const TorrentInfo &torrentInfo, const Path &folderPath, const QString &baseName);
+        void exportTorrentFile(const Torrent *torrent, const Path &folderPath);
```

Comment:
Some things for discussion:
Now there is `Torrent::exportToFile` and `Session::exportTorrentFile` and what are the differences to them?
I would imagine `Session::exportTorrentFile` should be exporting every torrent held in session and `Torrent::exportToFile` is for exporting a single torrent.

Reply:
> Now there is `Torrent::exportToFile` and `Session::exportTorrentFile` and what are the differences to them?

`Session::exportTorrentFile` is private helper. Currently it extends `Torrent::exportToFile` by adding unique file name generation.